src/Documentation/Getting-Started-With-Orleans/Grain-Persistence.md

```diff
@@ -134,6 +134,9 @@ A quick way to test this is to (see in the aforementioned the few commented line
 The ADO.NET persistence has functionality to version data and define arbitrary (de)serializers with arbitrary application rules and streaming, but currently
 there is no method to expose them to application code. More information in [ADO.NET Persistence Rationale](#ADONETPersistenceRationale).
 
+__NB for Microsoft SQL Server users:__
+For the ADO.NET Storage Provider to work with SQL Server, you need to add [System.Data.SqlClient](https://www.nuget.org/packages/System.Data.SqlClient/) NuGet package to your project.
```

Comment:
I also see the https://www.nuget.org/packages/Npgsql/ (PostgreSQL) being referenced in our tests project, nevertheless, no tests actually use it, so not sure if it works.
Also, there's these provider connection factories in our code: https://github.com/dotnet/orleans/blob/2c0cb3c11c86ee9b4e3299aedeeeab885a39825d/src/OrleansSQLUtils/Storage/DbConnectionFactory.cs#L23
Nevertheless, that doesn't mean they actually work, since there are no tests for them.
Probably just mentioning SQL Server and MySql packages is enough, and people can extrapolate from there.